<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>683</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLineEdit" name="additem_line_edit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>130</y>
     <width>471</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Poppins</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(236, 221, 214);</string>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Masukkan List Kegiatan</string>
   </property>
  </widget>
  <widget class="QListWidget" name="my_list">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>230</y>
     <width>471</width>
     <height>331</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>150</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Poppins</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="viewMode">
    <enum>QListView::ListMode</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="clear_all">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>180</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Poppins</family>
    </font>
   </property>
   <property name="text">
    <string>Clear List</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delete_item">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>180</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Poppins</family>
    </font>
   </property>
   <property name="text">
    <string>Hapus Item</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_item">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>180</y>
     <width>111</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Poppins</family>
    </font>
   </property>
   <property name="text">
    <string>Tambah Item</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ekspor">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>560</y>
     <width>141</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Poppins</family>
    </font>
   </property>
   <property name="text">
    <string>Export Excel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="impor">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>560</y>
     <width>141</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Poppins</family>
    </font>
   </property>
   <property name="text">
    <string>Import</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ekspor_pdf">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>600</y>
     <width>141</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Poppins</family>
    </font>
   </property>
   <property name="text">
    <string>Export PDF</string>
   </property>
  </widget>
  <widget class="QLabel" name="keterangan">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>210</y>
     <width>311</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Poppins</family>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#keterangan{
	color: rgb(255, 0, 0);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="exit">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>10</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>sit.png</normaloff>sit.png</iconset>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="update">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>180</y>
     <width>101</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Poppins</family>
    </font>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>70</y>
     <width>531</width>
     <height>581</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>20</y>
     <width>231</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Poppins</family>
     <pointsize>36</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>To Do List</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>711</width>
     <height>681</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(236, 221, 214);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>250</y>
     <width>471</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(236, 221, 214);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>280</y>
     <width>471</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(236, 221, 214);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>310</y>
     <width>471</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(236, 221, 214);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>340</y>
     <width>471</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(236, 221, 214);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_5">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>370</y>
     <width>471</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(236, 221, 214);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_6">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>400</y>
     <width>471</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(236, 221, 214);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_7">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>430</y>
     <width>471</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(236, 221, 214);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_8">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>460</y>
     <width>471</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(236, 221, 214);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_9">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>490</y>
     <width>471</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(236, 221, 214);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_10">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>520</y>
     <width>471</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(236, 221, 214);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <zorder>label_3</zorder>
  <zorder>label</zorder>
  <zorder>additem_line_edit</zorder>
  <zorder>my_list</zorder>
  <zorder>clear_all</zorder>
  <zorder>delete_item</zorder>
  <zorder>add_item</zorder>
  <zorder>ekspor</zorder>
  <zorder>impor</zorder>
  <zorder>ekspor_pdf</zorder>
  <zorder>keterangan</zorder>
  <zorder>exit</zorder>
  <zorder>update</zorder>
  <zorder>label_2</zorder>
  <zorder>line</zorder>
  <zorder>line_2</zorder>
  <zorder>line_3</zorder>
  <zorder>line_4</zorder>
  <zorder>line_5</zorder>
  <zorder>line_6</zorder>
  <zorder>line_7</zorder>
  <zorder>line_8</zorder>
  <zorder>line_9</zorder>
  <zorder>line_10</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
